github/resource_github_organization_project_test.go

```diff
@@ -0,0 +1,114 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/google/go-github/github"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccGithubOrganizationProject_basic(t *testing.T) {
+	var project github.Project
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccGithubOrganizationProjectDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGithubOrganizationProjectConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGithubOrganizationProjectExists("github_organization_project.test", &project),
+					testAccCheckGithubOrganizationProjectAttributes(&project, &testAccGithubOrganizationProjectExpectedAttributes{
+						Name: "test-project",
+						Body: "this is a test project",
+					}),
+				),
+			},
+		},
+	})
+}
+
```

Comment:
Do you mind adding a test for import also?

Reply:
Added test for import in ec08678. Thank you.